install.sh

```diff
@@ -0,0 +1,34 @@
+main() {
+
+	if which tput >/dev/null 2>&1; then
+      ncolors=$(tput colors)
+  fi
+  if [ -t 1 ] && [ -n "$ncolors" ] && [ "$ncolors" -ge 8 ]; then
+    RED="$(tput setaf 1)"
+    GREEN="$(tput setaf 2)"
+    NORMAL="$(tput sgr0)"
+  else
+    RED=""
+    GREEN=""
+    NORMAL=""
+  fi
+
+  set -e
+
+  echo "${GREEN}Installing DaVinci Video Converter..."
+  echo "${NORMAL}Cloning repository..."
+  git clone --depth=1 https://github.com/tpraxl/convert-footage || {
```

Comment:
I believe that most of the users don't have git installed. Please use curl or wget. 

Ideally, but not necessarily, check if curl is installed and fall back to wget otherwise.